.github/workflows/run-tests.yml

```diff
@@ -14,11 +14,11 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         php: [7.4, 8.0]
-        laravel: [8.*]
+        laravel: [8.*, 9.*]
```

Comment:
Considering that Laravel 9 requires PHP 8.0 as minimum I would also change the PHP targets to 8.0 and 8.1

I think is fine if we leave PHP 7.4 behind